### PR TITLE
Notifications for lists.

### PR DESCRIPTION
**Pull request by peteclark-ft**

### CODE REVIEW

File: list-notifications-push@.service [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   --memory="256m" \
+      --env "NOTIFICATIONS_RESOURCE=lists" \
+      --env "QUEUE_PROXY_ADDRS=http://%H:8080" \
+      --env "GROUP_ID=$GROUP_ID" \
+      --env "TOPIC=PostPublicationEvents" \
+      --env "NOTIFICATIONS_DELAY=$(/usr/bin/etcdctl get /ft/config/cache-max-age)" \
+      --env "QUEUE_HOST=kafka" \
+      --env "API_BASE_URL=$API_BASE_URL" \
+      --env "CONSUMER_BACKOFF=2" \
+      --env "WHITELIST=^http://.*-list-mapper\.svc\.ft\.com(:\d{2,5})?/lists/[\w-]+.*$" \
```
Comment (peteclark-ft) [inline list-notifications-push@.service]: @Nesze does this whitelist look ok to you?

Comment (Nesze) [inline list-notifications-push@.service]: yepp 👍

**Comment by Nesze** [comment]
Q: as it's the same codebase, shouldn't we release the same version of the notifications-push, too? I think that's the approach with the ingesters.

**Comment by peteclark-ft** [comment]
@Nesze yes we'll be upgrading content-notifications-push via another PR (as that has a dependency on PAM changes)

**Comment by khatton-ft** [comment]
Oh, just read @Nesze's comment.

**Pull request merged**